packages/components/src/Illustration/subcomponents/Base/Base.module.css

```diff
@@ -0,0 +1,97 @@
+@layer kz-components {
+  .wrapper {
+    width: 100%;
+    margin: 0;
+  }
+
+  .figure {
+    margin: 0;
+    position: relative;
+  }
+
+  .figure .pausePlayButton {
+    opacity: 0;
+    position: absolute;
+    left: 0;
+    bottom: 0;
+    border-width: 1px;
+    transition: all var(--animation-duration-immediate) ease;
+
+    @media (hover: none) and (pointer: coarse) {
+      background: var(--color-gray-200);
+      opacity: 1;
+    }
+  }
+
+  .figure .pausePlayButton:hover,
+  .figure .pausePlayButton:focus,
+  .figure:hover .pausePlayButton {
+    opacity: 1;
+    background: var(--color-gray-200);
+    border-color: var(--color-gray-600);
+  }
+
+  .hasVisibleAnimationToggle {
+    opacity: 1 !important;
+  }
+
+  .aspectRatioWrapper {
+    margin: 0;
+    display: flex;
+    align-items: center;
+  }
+
+  .landscape {
+    aspect-ratio: 4/3;
+
+    @supports not (aspect-ratio: auto) {
```

Comment:
I think we can safely use this without a fallback as it's a baseline widely avaiable feature and it landed 4+ years ago in most browsers